### ISSUE TITLE
Keep Enterprise Releases in vars

### ISSUE THREAD

**Issue by hnk** [alvistack/ansible-role-jira#7]
Hi there 👋 

We are using Jira Enterprise Releases in production. Currently I set those in the _jira_download array manually, but I was wondering if maybe other people would benefit if they are provided by the role itself?

Basically: Would you mind keeping up to date with ER information inside [the main vars file](https://github.com/alvistack/ansible-role-jira/blob/master/vars/main.yml)?

I can totally understand if you have no interest in keeping up to date on that info. Then I will just continue to add the info manually in our local repo :)

**Comment by barrybritt-vertex** [comment]
:+1: We are also using Enterprise Releases for Jira and would like to continue using your Ansible role.

**Comment by hswong3i** [comment]
TL;DR; Most likely only "-1 minor release" will be supported.

Currently I am keeping a "-1 minor release" support policy in most of my (80+) Ansible roles which manually download archive package from official site, e.g.
  - https://github.com/alvistack/ansible-role-jira/blob/master/vars/main.yml
  - https://github.com/alvistack/ansible-role-golang/blob/master/vars/main.yml 
  - https://github.com/alvistack/ansible-role-kubeadm/blob/master/vars/main.yml

The root cause of this is because for smooth upgrade: usually I just keep single version of templates file in role for latest version, where most likely they could be functioning across latest and "-1 minor release".

This is really tricky: I even just run CI test for different OS with **latest** release but not the "-1 minor release". Honestly, I am NOT guaranteeing latest code could working with "-1 minor release" but just blessing god for that :-P

In case of ER... e.g. JIRA now coming with ER in 8.5 but latest with 8.7... In order to secure the templates file functioning with 8.5 we may need:
  - Another SHA1 combo for 8.5 in `vars/main.yml`
  - Another set of templates, especially for 8.5 (e.g. https://github.com/alvistack/ansible-role-jira/blob/master/templates/opt/atlassian/jira/bin/setenv.sh.j2 and https://github.com/alvistack/ansible-role-jira/blob/master/templates/opt/atlassian/jira/bin/setenv.sh-8.5.3.j2)
  - A new logic in https://github.com/alvistack/ansible-role-jira/blob/master/tasks/main.yml so will care about the specific versioning when running template
  - Don't forget about Molecule + Travis...

That's also why I left that open for simple overwrite in `default/main.yml`: most likely you could control it with simple download info replacement, but sorry that I couldn't guarantee if it will be functioning for an outdated version :-S

**Comment by barrybritt-vertex** [comment]
That is fair.
I typically do use this feature, and I do not experience any issues with the role building out the setenv.sh file, so it may not be much of an issue for now (until atlassian changes something re: tomcat initialization).
Thanks for the info!
